packages/hooks/src/useInputDecimalFormatter.ts

```diff
@@ -12,7 +12,7 @@ const useInputDecimalFormatter = (initial?: number, options?: TOptions) => {
     const onChange = useCallback(
```

Comment:
Cognitive complexity reduced from 25 -> 23:
<img width="837" alt="Screenshot 2023-08-01 at 4 57 02 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/223534d6-6f0f-41d9-9bde-6ee30969793f">

Reply:
Update: Cognitive complexity has been reduced from 25 to 15 by introducing the boolean functions